### ISSUE TITLE
Add tracing support

### ISSUE THREAD

**Issue by XVilka** [dbousque/lymp#10]
To be able to trace Python code (including limp.py) with this module: https://docs.python.org/3.6/library/trace.html

**Comment by dbousque** [comment]
Hi @XVilka , you can certainly already use `trace` with lymp, since you can execute any arbitrary Python code using it, unless I'm missing something ? :)

**Comment by XVilka** [comment]
Yes, just modified the limp.py. Thought there are may be a better ways for seamless tracing/debugging.
